server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -407,7 +408,9 @@ private ClusterMetadataManifest uploadManifest(
                 Version.CURRENT,
                 nodeId,
                 committed,
-                uploadedIndexMetadata
+                uploadedIndexMetadata,
+                // todo Change this to proper cluster UUID
+                ClusterState.UNKNOWN_UUID
```

Comment:
We need to add UUID from previous cluster, right?

Reply:
Doing this in the follow up PR: https://github.com/opensearch-project/OpenSearch/pull/9831